CantoboardFramework/Keyboard/View/Status/StatusMenu.swift

```diff
@@ -83,15 +80,18 @@ class StatusMenu: UIView {
         super.layoutSubviews()
         layer.shadowPath = CGPath(roundedRect: bounds, cornerWidth: Self.cornerRadius, cornerHeight: Self.cornerRadius, transform: nil)
         
+        let rowHeight = bounds.height / CGFloat(itemLabelInRows.count)
+        let font = UIFont.systemFont(ofSize: bounds.width / CGFloat(itemLabelInRows[0].count) * Self.fontSizeRatio)
```

Comment:
What does `bounds.width / CGFloat(itemLabelInRows[0].count)` mean?
How is Self.fontSizeRatio calculated?